client/components/Timer.jsx

```diff
@@ -0,0 +1,134 @@
+/*
+    ./client/components/Timer.jsx
```

Comment:
If possible you can separate out Timer and Clock components into separate files.